cmd/krel/cmd/changelog.go

```diff
@@ -98,22 +100,21 @@ func init() {
 	)
 	changelogCmd.PersistentFlags().StringVar(&changelogOpts.bucket, "bucket", "kubernetes-release", "Specify gs bucket to point to in generated notes")
 	changelogCmd.PersistentFlags().StringVar(&changelogOpts.tag, tagFlag, "", "The version tag of the release, for example v1.17.0-rc.1")
-	changelogCmd.PersistentFlags().StringVar(&changelogOpts.tars, tarsFlag, "", "Directory of tars to SHA512 sum for display")
+	changelogCmd.PersistentFlags().StringVar(&changelogOpts.tars, tarsFlag, ".", "Directory of tars to SHA512 sum for display")
 	changelogCmd.PersistentFlags().StringVar(&changelogOpts.htmlFile, "html-file", "", "The target html file to be written. If empty, then it will be CHANGELOG-x.y.html in the current path.")
+	changelogCmd.PersistentFlags().StringVar(&changelogOpts.recordDir, "record", "", "Record the API into a directory")
+	changelogCmd.PersistentFlags().StringVar(&changelogOpts.replayDir, "replay", "", "Replay a previously recorded API from a directory")
 
 	if err := changelogCmd.MarkPersistentFlagRequired(tagFlag); err != nil {
 		logrus.Fatal(err)
 	}
-	if err := changelogCmd.MarkPersistentFlagRequired(tarsFlag); err != nil {
-		logrus.Fatal(err)
-	}
 
 	rootCmd.AddCommand(changelogCmd)
 }
 
 func runChangelog() (err error) {
 	token, ok := os.LookupEnv("GITHUB_TOKEN")
-	if !ok {
+	if !ok && changelogOpts.replayDir == "" {
 		return errors.New("environment variable `GITHUB_TOKEN` is not set but needed for release notes generation")
```

Comment:
How would you feel about updating this error to say either a Github token *or* a replayDir must be set?

Reply:
Yes sure, I update the error message :+1: